packaging/commands/interface.go

```diff
@@ -99,7 +99,6 @@ func NewPackageCommander(series string) (PackageCommander, error) {
 	default:
 		return NewAptPackageCommander(), nil
 	}
-	return nil, nil
```

Comment:
I am pretty sure this return is needed, I am not sure all versions of the compiler are that smart, what version did you try this with?